### ISSUE TITLE
easy api: botpress location message processor

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2067]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.20.4](https://github.com/open-wa/wa-automate-nodejs/compare/4.20.3...4.20.4) (2021-08-28)

- Release 4.20.4 [`5b04665`](https://github.com/open-wa/wa-automate-nodejs/commit/5b046652f4d6cbf8fedec8f70aca0a129a379ae9)
- ✨ EASY API: botpress integration can now handle location + media #2067 [`fc7ab0e`](https://github.com/open-wa/wa-automate-nodejs/commit/fc7ab0ede38eae2e118cd0028fcd9d54e32cdb97)
- 🐛 fix: #2068 [`2981de7`](https://github.com/open-wa/wa-automate-nodejs/commit/2981de73ed3992ab3d838c351606934df52c067b)
- updated types-only package [`da893db`](https://github.com/open-wa/wa-automate-nodejs/commit/da893db702b908eb44da85f6586bcf23bfd70573)

**Comment by m7mdcc** [comment]
Thanks , don't forget caption option for file message type
